tests/foreman/api/test_errata.py

```diff
@@ -199,17 +199,21 @@ def test_positive_install_multiple_in_host(
             'activationkey-id': ak.id,
         }
     )
-    # Install katello-ca, register content-host, enable all repos:
-    rhel_contenthost.install_katello_ca(target_sat)
-    rhel_contenthost.register_contenthost(function_org.name, ak.name, function_lce.name)
+    rhel_contenthost.register(
+        org=function_org,
+        activation_keys=ak.name,
+        lifecycle_environment=function_lce,
```

Comment:
when we've AK for registration then there's no need to LCE, and here your AK is created from function_lce already, so it can be removed too?
```suggestion
        activation_keys=ak.name,
```

Reply:
I will try without lce, the method `setup_org_for_a_custom_repo` still uses an lce passed or will create a new one